docs/maya/skin_merge.md

```diff
@@ -0,0 +1,78 @@
+# AdnSkinMerge
+
+AdnSkinMerge is a Maya deformer to blend animation and simulation together. It allows for the merging of several animation and simulation meshes into a single final mesh.
+
+The influence simulation or animation meshes will have on the final mesh can be freely painted and modified by painting a blend weights map.
+
+### How To Use
+
+AdnSkinMerge makes use of its own User Interface to create and modify the deformer.
+
+To create an AdnSkinMerge deformer within a Maya scene, the following inputs must be provided:
+
+  - **Final Mesh (F)**: Mesh to apply the merge results.
+  - **Animation Mesh List (Al)**: Mesh(es) to drive the simulation skin.
+  - **Simulation Mesh List (Sl)**: Mesh(es) with either an AdnSkin deformer applied or with the results from the skin simulation applied.
+
+The process to create an AdnSkinMerge deformer is:
+
+1. Press ![Skin merge button](images/adn_skin_merge.png){style="width:4%"} in the AdonisFX shelf or *Skin Merge* in the AdonisFX menu, under the *Deformers* submenu in the *Create* section to open the following UI.
+
+<figure markdown>
+  ![create skin merge UI](images/create_skin_merge.png) 
+  <figcaption><b>Figure 1</b>: Create Skin Merge UI.</figcaption>
+</figure>
+
+2. With the UI open select the final mesh in the scene and press the *Add Selected* button in the *Final Mesh* section.
```

Comment:
```suggestion
2. With the UI opened select the final mesh in the scene and press the *Add Selected* button in the *Final Mesh* section.
```

or simply

```suggestion
2. In the UI select the final mesh in the scene and press the *Add Selected* button in the *Final Mesh* section.
```

Reply:
Done! Applied the second suggestion, simple and more to the point.